crush.core

```diff
@@ -153,15 +219,41 @@ targets:
     toplevel: cpu_tb
     default_tool: icarus
 
+  sim_cpu_freertos:
+    <<: *default
+    filesets_append:
+      - sim_common
+      - sim_cpu_freertos
+    toplevel: cpu_freertos_tb
+    default_tool: icarus
+
+  sim_mtimer:
+    <<: *default
+    filesets_append:
+      - sim_common
+      - sim_mtimer
+    toplevel: mtimer_tb
+    default_tool: icarus
+
   sim_top:
     <<: *default
     filesets_append:
       - sim_top
+      - top_default
     toplevel: top_tb
     default_tool: icarus
 
+  sim_top_freertos:
+    <<: *default
+    filesets_append:
+      - sim_top_freertos
+      - sim_common
+      - top_freertos
+    toplevel: top_freertos_tb
+    default_tool: icarus
+
 scripts:
   iceprog_hint:
     cmd:
       - echo
-      - "Run 'iceprog ...' in the build folder to program the board"
+      - "Run 'iceprog -o 0x20000 freertos/app/build/crush_freertos.bin' to load firmware to flash."
```

Comment:
There should be two hints, one to program the FPGA bitstream, another to program the firmware.